src/services/bill_manager/bulk_invoice.rs

```diff
@@ -0,0 +1,72 @@
+use crate::client::{Mpesa, MpesaResult};
+use crate::constants::Invoice;
+use crate::environment::ApiEnvironment;
+use crate::errors::MpesaError;
+use serde::Deserialize;
+
+#[derive(Clone, Debug, Deserialize)]
+pub struct BulkInvoiceResponse {
+    #[serde(rename(deserialize = "rescode"))]
+    pub res_code: String,
```

Comment:
Just because safaricom use poor naming, doesn't mean we have to as well 😅

prefer the names in full for brevity